build-logic/integration-testing/src/main/kotlin/gradlebuild/integrationtests/tasks/GenerateAutoTestedSamplesTestTask.kt

```diff
@@ -99,7 +99,7 @@ class ${className} extends AbstractAutoTestedSamplesTest {
     '''
     @Test
     void runSamples() {
-        runSamplesFromFile(new File('${file.file.absolutePath.replace("\\", "/")}'), FILE_CONTENT)
+        runSamplesFromFile(new File('${file.relativePath.toString().replace("\\", "/")}'), FILE_CONTENT)
```

Comment:
☑️ assuming that `relativePath` is relative to the appropriate directory so `runSamplesFromFile` can locate it